devices/explosivebarrel.lua

```diff
@@ -22,6 +22,7 @@ IncendiaryRadiusHeated = 150
 StructureSplashDamage = 150
 StructureSplashDamageMaxRadius = 100
 IgnitePlatformOnDestruct = true
+BuildTimeComplete = 30
```

Comment:
Why is this "BuildTimeComplete" in the device.lua? It has no effect there, you want to put it in the device_list.lua

Reply:
it doesn't work when in the device list. I'll check though

Reply:
Yep, you were completely correct. I'm moving this into the device list.